src/targets/common.ini

```diff
@@ -80,7 +82,10 @@ board = bluepill_f103c8
 build_unflags = -Os
 build_flags =
 	-D PLATFORM_STM32=1
+	-Wl,-Map,firmware.map
 	-O2
+	-I ${PROJECTSRC_DIR}/hal
 src_filter = ${common_env_data.src_filter} -<ESP32*.*> -<ESP8266*.*> -<WS281B*.*>
 lib_deps =
 	paolop74/extEEPROM @ ^3.4.1
+	HAL
```

Comment:
Not needed

Reply:
oops, typo. Will be removed. thanks!

Reply:
Yeah, I like the way you've done it actually, so just delete this one. Then I think it's good to go unless there are changes required for lto.

Reply:
fixed